app/server/utils/user.py

```diff
@@ -219,7 +219,7 @@ def save_device_info(device_info, user):
     if device_info['serialNumber'] and not DeviceInfo.query.filter_by(serial_number=device_info['serialNumber']).first():
         # Add the device if the serial number is defined, and isn't already in db
         add_device = True
-    elif not device_info['serialNumber'] and not DeviceInfo.query.filter_by(unique_id=device_info['uniqueID']).first():
+    elif not device_info['serialNumber'] and not DeviceInfo.query.filter_by(unique_id=device_info['uniqueId']).first():
```

Comment:
@enjeyw Mobile still passes param as `uniqueID`. Need to update.

https://github.com/teamsempo/SempoMobileApp/blob/2d8f42b3e40ba173a6929b2c47838d3e2a7e1c1a/src/sagas/authSaga.js#L144